src/script/util/ValidationUtil.ts

```diff
@@ -72,31 +72,31 @@ export const isBase64 = (string: string) => {
   return true;
 };
 
-export const isValidApiPath = (path: string) => {
+export const isValidApiPath = (path: string): boolean => {
   const [urlPath] = path.split('?');
   if (!/^\/[a-zA-Z0-9\-_/,]+$/.test(urlPath)) {
     throw new ValidationUtilError(`Non-compliant path creation attempt. Details: ${path}`);
   }
   return true;
 };
 
-export const isTweetUrl = (url: string) => {
+export const isTweetUrl = (url: string): boolean => {
   const regex = /^http(?:s)?:\/\/(?:(?:www|mobile|0)\.)?twitter\.com\/(?:(?:\w{1,15})\/status(?:es|\/i)?|i\/moments)\/(?:\d{2,21})(?:(?:\?|\/).*)?$/;
   return regex.test(url);
 };
 
-export const legacyAsset = (assetId: string, conversationId: string) => {
+export const legacyAsset = (assetId: string, conversationId: string): true => {
```

Comment:
that's an interesting one :)

Reply:
Yeah, returning `true` or throwing an error is a bit useless but I didn't want to change it's behavior. :)